### ISSUE TITLE
Jaws spell doesn't work indoors

### ISSUE THREAD

**Issue by ferderber** [matthewferderber/Magico#13]
When used indoors, the spell will place all of the "Jaws" on the top side of the building (if the building is < 5 in height)

May be better to raycast on the ground to prevent this and other "climbing jaw" bugs
